src/features/layout/Modal.module.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../common/colors";
+
+dialog.modal {
+  $padding: 1em;
+  $border: 1px solid use-color("base-light");
+  border: $border;
+  display: flex;
+  flex-flow: column nowrap;
+  overflow: visible;
+  padding: $padding;
+
+  .header {
+    align-items: start;
```

Comment:
I think this is what the build is complaining about. It wants you to use `flex-start` instead.

Reply:
funky. I wonder if there is a linter setting given the warning is for browsers we don't support.

Reply:
Turns out it is not the linter (since _the linter is not running_!) so I am not totally sure where to tell it to chill. For the purposes of this PR I'm ok with this being the only problem. See #102 for other exciting fixes like: actually run the linter.